otherlibs/unix/socket.c

```diff
@@ -27,9 +27,9 @@ int socket_domain_table[] = {
   PF_UNIX, PF_INET,
 #if defined(HAS_IPV6)
   PF_INET6
-#elif defined(PF_UNDEF)
-  PF_UNDEF
-#else
+#elif defined(PF_UNSPEC)
+  PF_UNSPEC
+#elif
```

Comment:
This should be `#else`, not `#elif`, which is not valid C !

Reply:
Sorry about that; it's fixed now.